concert/devices/cameras/base.py

```diff
@@ -108,12 +103,17 @@ class Camera(Device):
     frame_rate = Quantity(1 / q.second, help="Frame frequency")
     trigger_source = Parameter(help="Trigger source")
     _senders: Dict[CommData, ZmqSender]
+    mirror = Parameter(help="Mirror the image")
+    rotate = Parameter(help="Rotate the image")
```

Comment:
Why not _one_ parameter with choices:
```python
convert = Choices("fliplr", "flipud", "rot90", ...)
```

Reply:
If we keep adding things we will have maaaaany parameters.

Reply:
Mirror true/false + 0-3 rotations are all combinations. I have to add all 8 combinations then as choices.

Reply:
Right. I mean if  you promise me that you and no one else will want to keep adding weird transformations... :smiley: